### DISCUSSION TITLE
如何在unocss中修改elementplus组件的内部样式

### DISCUSSION THREAD

**Discussion by dinghui-wang** [element-plus/element-plus#18188] [Q&A]
<img width="627" alt="Snipaste_2024-09-05_16-47-59" src="https://github.com/user-attachments/assets/c331c9b1-5f62-4f67-993b-e453c3522df3">
el-input的插槽有el-input-group__prepend属性，在unocss中如何修改里面的属性

**Comment by warmthsea** [comment]
参考：https://github.com/element-plus/element-plus/issues/17392
深层样式需要 :deep()

**Reply by dinghui-wang** [reply]
给个示例，:deep在哪儿写？style标签里面，还是template的上html标签上？

**Reply by warmthsea** [reply]
unocss 似乎是修改不了内部的深层样式的，只能通过
```scss
<style scoped>
  :deep(.el-input-group__prepend){
    // ...
  }
</style>
```

**Reply by dinghui-wang** [reply]
ok，实现了
<img width="326" alt="Snipaste_2024-09-05_18-32-58" src="https://github.com/user-attachments/assets/7b9afd35-ded4-4592-88a6-be8213ac714d">
